content/python/concepts/built-in-functions/terms/float/float.md

```diff
@@ -14,31 +14,33 @@ CatalogContent:
   - 'paths/data-science'
 ---
 
-Takes in a numeric, string, or no value and a copy of that value in the floating point data type.
+The built-in `float()` function returns a float value based on a [string](https://www.codecademy.com/resources/docs/python/strings), numeric [data type](https://www.codecademy.com/resources/docs/python/data-types), or no value at all.
 
 ## Syntax
 
-```py
-float()
+```pseudo
+float(num_string)
 ```
 
-## Example 1
+The `num_string` parameter is optional and should either be string or numeric type.
 
-Use `float()` to change the datatype of an integer value `314` to float:
+## Example
 
-```codebyte/python
+In the example, the `float()` function is used to return float-type versions of an integer value `314` and a string "314":
+
+```py
 print(float(314))
+print(float("314"))
 ```
 
-## Example 2
-
-Use `float()` to change the datatype of a string value `"314"` to float:
+The following output will look like this:
 
-```codebyte/python
-print(float("314"))
+```shell
+314.0
+314.0
 ```
 
-## Example 3
+## Codebyte Example
 
 Use `float()` to create a new float value:
```

Comment:
Not sure how much value there is in keeping this last example. This isn't really demonstrating any functionality that isn't already covered above